package.json

```diff
@@ -143,6 +143,7 @@
     "cypress-tags": "^0.0.20",
     "cypress-terminal-report": "^2.4.0",
     "cypress-wait-until": "^1.7.1",
+    "elliptic": "^6.5.4",
```

Comment:
is this being used anywhere?

Reply:
I don't think we use it, but other packages list it as a dependency in `package-lock.json`, so we can probably remove it from `package.json`...